test/e2e/flask/user-operations.spec.ts

```diff
@@ -249,16 +97,13 @@ async function withAccountSnap(
 describe('User Operations', function () {
   it('from dApp transaction', async function () {
     await withAccountSnap({ title: this.test?.fullTitle() }, async (driver) => {
-      const transaction = {
+      await createDappTransaction(driver, {
         from: ERC_4337_ACCOUNT,
         to: LOCAL_NODE_ACCOUNT,
         value: convertETHToHexGwei(1),
         maxFeePerGas: '0x0',
         maxPriorityFeePerGas: '0x0',
-      };
-      await driver.openNewPage(
-        `${DAPP_URL}/request?method=eth_sendTransaction&params=${JSON.stringify([transaction])}`,
-      );
+      });
 
       await confirmTransaction(driver);
```

Comment:
- we either use this step by step,
- we create a new flow
- we add this in the user-operations flow, instead of using that selector
```suggestion
      const confirmation = new Confirmation(driver);
      await confirmation.checkPageIsLoaded();
      await confirmation.checkPageIsLoaded();
      await confirmation.clickFooterConfirmButtonAndAndWaitForWindowToClose();
```